src/osd/ReplicatedBackend.cc

```diff
@@ -1572,8 +1572,8 @@ void ReplicatedBackend::prep_push(
   pi.recovery_info.version = version;
   pi.recovery_progress.first = true;
   pi.recovery_progress.data_recovered_to = 0;
-  pi.recovery_progress.data_complete = 0;
-  pi.recovery_progress.omap_complete = 0;
+  pi.recovery_progress.data_complete = false;
```

Comment:
@TsaiJin i think we can just remove following lines. as they are already initialized by the default ctor of `ObjectRecoveryProgress`
```c++
  pi.recovery_progress.first = true;
  pi.recovery_progress.data_recovered_to = 0;
  pi.recovery_progress.data_complete = 0;
  pi.recovery_progress.omap_complete = 0;
```